src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -65,7 +65,7 @@ public function index(Request $request)
             // If a column has a relationship associated with it, we do not want to show that field
             $this->removeRelationshipField($dataType, 'browse');
 
-            if ($search->value && $search->key && $search->filter) {
+            if ((!empty($search->value) || $search->value === '0') && $search->key && $search->filter) {
```

Comment:
This will result in the same and is much more generic
```suggestion
            if ($search->value != '' && $search->key && $search->filter) {
```